src/app.rs

```diff
@@ -331,12 +330,28 @@ fn initialize_window(
         gtk::Window::new(gtk::WindowType::Popup)
     };
 
-    window.set_title(&format!("Eww - {}", window_def.name));
-    let wm_class_name = format!("eww-{}", window_def.name);
-    window.set_wmclass(&wm_class_name, &wm_class_name);
+    #[cfg(feature = "wayland")]
+    {
+        // Inititialising a layer shell surface
+        gtk_layer_shell::init_for_window(&window);
+        // Sets the monitor where the surface is shown
+        match window_def.screen_number {
+            Some(index)=>{
+                let monitor = get_monitor(index);
+                gtk_layer_shell::set_monitor(&window, &monitor);
+            },
+            None=>{}
+        };
+        gtk_layer_shell::set_keyboard_interactivity(&window, window_def.focusable);
+    }
```

Comment:
formatting REEEEEEEEEEEEEEEEEEEEEEEEEEEE
But also, can we put this into a function somehow? Just having a big cfg(feature...) block in the middle of the function is kinda ehhhhhhhhhh. a `wayland_initialize_window` thing or sth may be reasonable